packages/dashmate/src/listr/tasks/buildServicesDepsTaskFactory.js

```diff
@@ -0,0 +1,39 @@
+const { Listr } = require('listr2');
+
+/**
+ *
+ * @param {DockerCompose} dockerCompose
+ * @param {Docker} docker
+ * @return {buildServicesDepsTask}
+ */
+function buildServicesDepsTaskFactory(
```

Comment:
Won't be simpler to put move this task to buildServicesTask. They won't be used separately